holoviews/plotting/mpl/element.py

```diff
@@ -698,13 +698,17 @@ class ColorbarPlot(ElementPlot):
     symmetric = param.Boolean(default=False, doc="""
         Whether to make the colormap symmetric around zero.""")
 
+    extend = param.ObjectSelector(
```

Comment:
I think this should probably be `cbar_extend` or `colorbar_extend` since ``extend`` could mean a number of things.